pkgs/by-name/li/libui-ng/package.nix

```diff
@@ -47,7 +47,9 @@ stdenv.mkDerivation rec {
     (lib.mesonBool "examples" (!stdenv.isDarwin))
   ];
 
-  passthru.updateScript = unstableGitUpdater { };
+  passthru.updateScript = unstableGitUpdater {
+    tagPrefix = "alpha";
```

Comment:
```suggestion
    # soversion is set to 0, and only alpha versions are available
    hardcodeZeroVersion = true;
```

Reply:
Counter point: This is a fork of https://github.com/andlabs/libui, where the tags come originate from. We have packaged that one by using the numerical part of the version, without accounting for any alpha-ness (I disagree with the way the alpha was working into that version, but w/e):

https://github.com/NixOS/nixpkgs/blob/e9be42459999a253a9f92559b1f5b72e1b44c13d/pkgs/development/libraries/libui/default.nix#L9-L15

Can hardcode zero if you think that makes more sense here. Neither of those packages has a maintainer, otherwise I'd ask for their opinion.

Reply:
Okay, let's leave as-is, then.